app/views/users/show.html.erb

```diff
@@ -1,2 +1,56 @@
-<h1>Hi, this is the profile of a user</h1>
 
+
+<div class="d-flex row justify-content-center mt-5">
+  <div class= "card text-dark bg-light  p-0 m-1" style="max-width: 7rem" >
+      <div class=" border-end pr-2">
+        <%= image_tag("images.png", :alt => "rss feed", class: 'img-fluid rounded-start')   %>
+      </div>
+  </div>
+
+  <div class="card text-dark bg-light m-1 d-flex flex-row col-md-8">
+    <div class="col-md-4  d-flex align-items-center">
+      <h5 class="card-title"> <%= link_to @user.name, user_url(@user), class: 'display-5 text-primary' %></h5> 
+    </div>
+
+    <div class = " col-md-7  d-flex align-items-end justify-content-end" >
+      <p class="text-right">Number of posts: 
+        <% @user.posts_counter ?  @user.posts_counter : 0   %>
+        </p>
+    </div>  
+  </div>
+
+    <div class = "card text-white bg-info col-md-9 m-1 d-flex" >
+      <h1>Bio:</h1>
+     <p><%= @user.bio   %></p>
+    </div>
+
+    <div class = "col-md-9 m-1  p-3 d-flex" >
+      <h2><%= @user.name%>'s 3 recent posts: </h2>
+    </div>
+
+    <% @user.three_recent_post.each do |post| %>
+      <div class = "  card col-md-9 m-1  p-3 d-flex" >
+      <% if post.comments_counter %>
+        <a href="/users/<%= @user.id %>/posts/<%= post.id %>"><p class="m-0"><h5> Post# <%=  post.id %> </h5></a>
+      <% else %>  
+        <h5> Post# <%=  post.id %> </h5>
```

Comment:
- [ ] Kindly delete commented codes if the code after `Post` is commented out. Do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history.

Reply:
This hs been resolved.